src/react-components/src/components/Modal/Modal.stories.tsx

```diff
@@ -1,14 +1,52 @@
 import * as React from "react";
 
+import Container from "../Container/Container";
+import Button from "../Button/Button";
 import Modal from "./Modal";
 
 export default {
     title: "Modal",
     component: Modal,
 };
 
-export const modal = () => (
-    <Modal>
-        <div>Modal Content</div>
-    </Modal>
-);
+class ModalStory extends React.Component<{}, { isModalOpen: boolean }> {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isModalOpen: false,
```

Comment:
Should the default for the modal be open?

Reply:
I'm agnostic /shrug 

Whichever tells the story best! We can switch it if you the story being in the other direction